pyiceberg/catalog/sql.py

```diff
@@ -673,11 +673,16 @@ def update_namespace_properties(
                     IcebergNamespaceProperties.property_key.in_(set(updates.keys())),
                 )
                 session.execute(delete_stmt)
-                insert_stmt = insert(IcebergNamespaceProperties)
-                for property_key, property_value in updates.items():
-                    insert_stmt = insert_stmt.values(
-                        catalog_name=self.name, namespace=namespace_str, property_key=property_key, property_value=property_value
-                    )
+                insert_stmt_values = [
+                    {
+                        "catalog_name": self.name,
```

Comment:
nit: instead of the `"catalog_name"` string, might be able to substitute with the `IcebergNamespaceProperties.catalog_name` field.